src/cdk/drag-drop/directives/drag.ts

```diff
@@ -139,6 +141,9 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
   /** Class to be added to the preview element. */
   @Input('cdkDragPreviewClass') previewClass: string | string[];
 
+  /** Target for drag clone to append into */
+  @Input('cdkDragCloneTarget') cloneTarget: string | ElementRef<HTMLElement> | HTMLElement;
```

Comment:
Oh I really like that!
I'll look into it over the weekend

Reply:
`global`, `ElementRef` and `HTMLElement` are not a problem, however having it alongside it's self is causing major perf problems on drag.

I *think* this is caused by the inline style `transform` triggering "changes" in the `DragList`. As such i've appended it to the parent of the `DragList`, hope this is ok?

Reply:
What sorts of performance issues is it causing? I wouldn't expect an extra style mutation to have much of an effect since we aren't using any mutation observers and the DOM is only checked once dragging has been initiated.

I think that `parent` should mean "next to the current `cdkDrag`" since the `cdkDropList` could have some styles that are inherited to the child elements.

Reply:
Specifically its render delays in the preview element:
- Dragging becomes slow, it takes a while for the drag preview item to catch up with the mouse position
- It appears to be impacting CSS selectors from cascading correctly.

In the demo app there's a selector `.demo-list .cdk-drag-preview` which applies a border when dragging but only when the preview container is set to parent, however when i'm setting the drag container to be the same parent as the `cdkDrag` element, the styles are't getting applied.

When dragging the item out of the list into another and back in, then it's getting applied, i'm assuming that works because of the re-render caused by `transferArrayItem`.

Any advice here would be more than welcomed

Reply:
Hey @crisbeto - I'm about to push all the other changes we've discussed but i still dont have a conclusion on this point, i can make `parent` work as described but it's still a concern that there's serious performance issues when doing it. I've not been able to even diagnose what's causing it.

Presently thinking we could either:
- name it such as `list-sibling` to better describe where it's going
- remove any concep of `parent` and leave it as `global` or a specified location?

Reply:
If you feel that `parent` is problematic, we can leave it out of this PR and add it in a follow-up. That being said, wouldn't it still perform in the same way if you passed in the parent as a raw DOM node?

Reply:
I'd appreciate that, i guess if we could agree some wording then it might be worth mentioning in the docs.
Something like: "Specifying a target inside the `DragList` can lead to rendering performance issues" ?